README.md

```diff
@@ -30,8 +41,5 @@ and re-use in subsequent research and analysis.
 
 ## Dependencies
 
-The data and notebooks in this repository use the following packages: 
-
- - [pyam](https://pyam-iamc.readthedocs.io)
-   (note that this package is distributed on pypi as `pyam-iamc`)
- - [scmdata](https://scmdata.readthedocs.io) 
+The data and notebooks in this repository use the dependencies specified in `environment.yml`.
```

Comment:
I would actually keep both the high-level overview of the key packages that are required, and then add the technical instructions on how to install it...

Reply:
Ok nice will put back in